package-lock.json

```diff
@@ -0,0 +1,5893 @@
+{
```

Comment:
That one slipped through, not deliberate, thanks for catching.  For now I have put it in the .gitignore.